src/js/ui/LottoGameViewWeb.js

```diff
@@ -0,0 +1,130 @@
+import LottoGameView from './LottoGameView.js'
+
+class LottoGameViewWeb extends LottoGameView {
+  #restart
+
+  constructor() {
+    super()
+    this.#restart = 'n'
+  }
+
+  async getPurchaseAmount() {
+    return new Promise(async (resolve, reject) => {
+      const button = document.getElementById('purchase-button')
+
+      button.addEventListener('click', async () => {
+        const input = document.getElementById('purchase-amount-input')
+        const purchaseAmount = input.value
+        resolve(purchaseAmount)
+      })
+    })
+  }
+
+  async getLottoWinningNumbers() {
+    const { selectedNums, extraNum } = await this.#createButtonClickPromise()
+    return { selectedNums: selectedNums, extraNum }
+  }
+
+  printPurchasedLottos(purchasedLottoList) {
+    const lottoAmount = purchasedLottoList.length
+
+    // 구매한 갯수 출력
+    const view = document.getElementById('purchased-amount-view')
+    const message = `총 ${lottoAmount}개를 구매하였습니다.`
+    view.textContent = message
+
+    // 로또 이미지, 로또 번호 출력
+    const lottoWrapper = document.getElementById('ticket-image-wrapper')
+    purchasedLottoList.forEach((lotto) => {
+      const { selectedNums, extraNum } = lotto.numbers
+      const numbers = [...selectedNums, extraNum].sort((a, b) => a - b)
+
+      const ticketWrapper = document.createElement('div')
+      ticketWrapper.className = 'd-flex items-center'
+
+      const ticketElement = document.createElement('span')
+      ticketElement.className = 'mx-1 text-4xl'
+      ticketElement.textContent = '🎟️'
+
+      const numberElement = document.createElement('span')
+      numberElement.className = 'mx-1 text-base'
+      numberElement.textContent = numbers.join(', ')
```

Comment:
취향 차일 수 있긴한데요~
선언을 위로 분리해보는 것도 추천드립니다 ㅎㅎ

```suggestion
      const ticketWrapper = document.createElement('div')
      const ticketElement = document.createElement('span')
      const numberElement = document.createElement('span')
            
      ticketWrapper.className = 'd-flex items-center'
      
      ticketElement.className = 'mx-1 text-4xl'
      ticketElement.textContent = '🎟️'

      numberElement.className = 'mx-1 text-base'
      numberElement.textContent = numbers.join(', ')
```